qirlib/src/context.rs

```diff
@@ -10,23 +10,26 @@ use crate::{
     types::Types,
 };
 
-pub struct BareContext<'ctx> {
+pub struct Bare<'ctx> {
```

Comment:
I think having "bare context" and "context" made sense, and you can guess the relationship between them, but not with "bare" and "context."

Reply:
I agree. We need a better name. Clippy complained about the `Context` suffix. We can change it back but we'll have to attribute it.